proto/stakeibc/tx.proto

```diff
@@ -65,6 +65,7 @@ message MsgRegisterHostZone{
   string base_denom = 4 [ (gogoproto.moretags) = "yaml:\"base_denom\"" ];
```

Comment:
Nit: 

This PR feels like good PR to change the denom names from "base" and "local" to something more intuitive. I still every time I see them have trouble remembering which is which with the current names. 

Maybe baseDenom (e.g. uatom) and stDenom (stuatom)?

Reply:
Also agreed these aren't intuitive but probably better to make this an "urgent fix" after the sprint compiles? I'll add a ticket for it right now